test/unit/version_test.lua

```diff
@@ -0,0 +1,8 @@
+local t = require("luatest")
+local g = t.group('expirationd_versioning')
+local expirationd = require('expirationd')
+
+g.test_version = function()
+    t.assert_type(expirationd._VERSION, 'string')
+    t.assert(string.find(expirationd._VERSION, "^%d+%.%d+%.%d+$"))
```

Comment:
Since `find` doesn't return boolean, I think it's better to do assertion without implicit convertions.

Reply:
if I understood correctly, mean the non-nil `find` return check?

Reply:
For example

Reply:
Oh, I see, it is already replaced to `t.assert_not_equals(string.find(expirationd._VERSION, "^%d+%.%d+%.%d+$"), nil)`
Could you do the same for `etcd-client`, please?

Reply:
Oh, I see it is in `etcd-client` already =) Thank you!